spec/index.bs

```diff
@@ -14688,3 +14691,26 @@ Eventually all of these should disappear but they are useful to avoid warning wh
 [=Origin2D/x=] [=Origin2D/y=]
 [=RenderPassDescriptor/renderExtent=]
 [=vertex buffer=]
+
+<script>
+// Small script to collect all xrefs that refer to different timelines and highlight
+// them with an appropriate style.
+
+const dataLinks = document.querySelectorAll('[data-link-type]');
+for (const dataLink of dataLinks) {
+    const linkUrl = dataLink.getAttribute('href');
+    if (linkUrl && linkUrl.startsWith('#')) {
+      const dataLinkTarget = document.getElementById(linkUrl.substring(1));
+      if (dataLinkTarget) {
+          // Find the closets ancestor of the target (including the target itself)
+          // that contains a data-timeline.
+          const timelineElement = dataLinkTarget.closest('[data-timeline]');
+          if (timelineElement) {
+            // If we found a timeline, apply an appropriate style the link.
+            const timeline = timelineElement.getAttribute('data-timeline');
+            dataLink.setAttribute('data-timeline', timeline);
+          }
+      }
+    }
+}
+</script>
```

Comment:
In my local changes for timeline stuff, ends up applying a style to GPUBufferMapState and all its enum-values because it's defined inside a timeline. Maybe we only want this to apply to specific dfn types, like, `abstract-op`, `attribute`, and `dfn`?